package.json

```diff
@@ -4,32 +4,30 @@
   "version": "0.4.5",
   "main": "src/jest.js",
   "dependencies": {
-    "coffee-script": "^1.8.0",
-    "cover": "~0.2.8",
-    "diff": "~1.0.4",
-    "graceful-fs": "^2.0.3",
-    "harmonize": "1.4.2",
-    "istanbul": "^0.3.2",
-    "jasmine-only": "0.1.0",
-    "jasmine-pit": "~2.0.0",
-    "jsdom": "~0.10.3",
+    "coffee-script": "ide/coffeescript#array-check",
```

Comment:
Would be nice having this merged to upstream, and to use the new version of CoffeeScript.